src/backend/model/data/migrations/20220803030004_create_menus_table.js

```diff
@@ -0,0 +1,25 @@
+const TABLE_NAME = "menus";
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.createTable(TABLE_NAME, (table) => {
+    table.increments("id").primary();
+    table.integer("shop_id").notNullable();
+    table.foreign("shop_id").references("shops.id");
+    table.string("name", 255).notNullable();
+    table.integer("price").notNullable();
```

Comment:
Probably should be a float with 2 decimal points

Reply:
Is price have floating points? I think it always be integer.